signer/src/storage/model.rs

```diff
@@ -536,9 +533,8 @@ pub struct EncryptedDkgShares {
     pub started_at_bitcoin_block_hash: BitcoinBlockHash,
     /// The block height of the chain tip of the canonical bitcoin blockchain
     /// when the DKG round associated with these shares started.
-    #[sqlx(try_from = "i64")]
     #[cfg_attr(feature = "testing", dummy(faker = "0..i64::MAX as u64"))]
-    pub started_at_bitcoin_block_height: u64,
+    pub started_at_bitcoin_block_height: BitcoinBlockHeight,
```

Comment:
We should no longer need the `cfg_attr` here.

Reply:
Thanks!

Reply:
[fa39d1e](https://github.com/stacks-sbtc/sbtc/pull/1448/commits/fa39d1e500d7df870afaa533ae8dee34cc19dbdf)